RFC/src/RFC-0202_TariScriptOpcodes.md

```diff
@@ -325,6 +325,23 @@ push 0.
 Identical to [`CheckSig`](#checksigmsg), except that nothing is pushed to the stack if the signature is valid, and the
 operation fails with `VERIFY_FAILED` if the signature is invalid.
 
+##### CheckMultiSig(Msg)
+
+Pop the number $n$ indicating the number of allowed spenders, and then pop $n$ spender public keys off the
+stack. Then pop the number $m$ indicating the number of required spending signatures, and then pop $m$
+signatures from the stack provided as inputs to the script. If $m$ signatures of the possible $n$ public keys
```

Comment:
It feels like there are too many 'then`, `and then` sequences in these first two sentences.

Reply:
Open to suggestions

Reply:
This is fine. No need to change it

Reply:
If we feel like it reads too difficult, perhaps make it a numbered list.

Reply:
This needs to be changed to what was merged in tari-crypto anyway.